code/game/objects/items/cosmetics.dm

```diff
@@ -22,6 +28,11 @@
 	if(vname == NAMEOF(src, open))
 		update_appearance(UPDATE_ICON)
 
+/obj/item/lipstick/examine(mob/user)
+	. = ..()
+
+	. += "Alt-click on [src] to change the style."
```

Comment:
```suggestion
	. += "Alt-click to change the style."
```
"This is red lipstick. Alt click on the red lipstick to change the style."
vs
"This is red lipstick. Alt click to change the style."